integration/go.mod

```diff
@@ -18,4 +18,5 @@ require (
 	goharvest2 v0.0.0-00010101000000-000000000000
 	golang.org/x/time v0.0.0-20210723032227-1f47c861a9ac // indirect
 	google.golang.org/grpc v1.39.1 // indirect
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
```

Comment:
go version should be same as harvest go version in this file.

Reply:
will check and fix it